tmail-backend/guice/distributed/src/main/java/com/linagora/tmail/ScheduledReconnectionHandler.java

```diff
@@ -300,7 +300,10 @@ class QueueNotFoundException extends RuntimeException {
         "JamesMailQueue-workqueue-outgoing",
         "mailboxEvent-workQueue-org.apache.james.events.GroupRegistrationHandler$GroupRegistrationHandlerGroup",
         "jmapEvent-workQueue-org.apache.james.events.GroupRegistrationHandler$GroupRegistrationHandlerGroup",
-        "deleted-message-vault-work-queue");
+        "deleted-message-vault-work-queue",
+        "openpaas-contacts-queue-add",
+        "openpaas-contacts-queue-update",
+        "openpaas-contacts-queue-delete");
```

Comment:
the issue arises when the extra queues are added here.
in `SchedulerReconnectionHandlerIntegrationTest.java`, the tests expect each queue to have consumers, but these do not have any, and i don't know how to add them.
i tried to add a `OpenPaasContactsConsumerProbe.java` but it seems like it does not change much.